### DISCUSSION TITLE
File Explorer opened with arguments error

### DISCUSSION THREAD

**Discussion by Jamesstevens-3rd** [valinet/ExplorerPatcher#2152] [General]
on windows 11 when the file explorer is started with any argument from a shortcut or CLI it will open with Window 11 Command Bar irrespective of whether Control Interface is set to Windows 10 Ribbon or Windows 7 Command Bar

list of switches
https://ss64.com/nt/explorer.html

examples:

%SystemRoot%\explorer.exe /e,E:

Explorer \\

explorer.exe c:

Explorer.exe "C:\Intel"

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by Jamesstevens-3rd** [reply]
Windows 11 Pro 22H2 22621.2283

Explorer Patcher 22621.2283.57.2

`Windows Registry Editor Version 5.00

[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ImportOK"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"OldTaskbar"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_TaskbarPosition"=dword:00000003
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_MMTaskbarPosition"=dword:00000003
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Search]
"SearchboxTaskbarMode"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"ShowTaskViewButton"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_AutoHideTaskbar"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"OrbStyle"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"OldTaskbarAl"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"MMOldTaskbarAl"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"TaskbarGlomLevel"=dword:00000002
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"MMTaskbarGlomLevel"=dword:00000002
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarSmallIcons"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"SkinMenus"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"CenterMenus"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"FlyoutMenus"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\TabletTip\1.7]
"TipbandDesiredVisibility"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"ShowSecondsInSystemClock"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"HideControlCenterButton"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarSD"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ReplaceNetwork"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Control Panel\Settings\Network]
"ReplaceVan"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows NT\CurrentVersion\MTCUVC]
"EnableMtcUvc"=dword:00000001
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\ImmersiveShell]
"UseWin32TrayClockExperience"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\ImmersiveShell]
"UseWin32BatteryFlyout"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"IMEStyle"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_RegisterAsShellExtension"=dword:00000000
[HKEY_CURRENT_USER\Software\Classes\CLSID\{86ca1aa0-34aa-4e8b-a509-50c905bae2a2}\InprocServer32]
@="-"
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"LegacyFileTransferDialog"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UseClassicDriveGrouping"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_FileExplorerCommandUI"=dword:00000002
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableImmersiveContextMenu"=dword:00000000
[-HKEY_CURRENT_USER\Software\Classes\CLSID\{056440FD-8568-48e7-A632-72157243B55B}\InprocServer32]
@=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_DisableModernSearchBar"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ShrinkExplorerAddressBar"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"HideExplorerSearchBar"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"Start_ShowClassicMode"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"TaskbarAl"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_Start_MaximumFrequentApps"=dword:00000006
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StartPage]
"MonitorOverride"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_StartDocked_DisableRecommendedSection"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StartPage]
"MakeAllAppsDefault"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer]
"AltTabSettings"=dword:00000003
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_PeopleBand"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"LastSectionInProperties"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ClockFlyoutOnWinC"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ToolbarSeparators"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"PropertiesInWinX"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"NoMenuAccelerator"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableOfficeHotkeys"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableWinFHotkey"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
;"Virtualized_{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_DisableRoundedCorners"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DisableAeroSnapQuadrants"=dword:00000000
[HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced]
"Start_PowerButtonAction"=dword:00000002
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectSystemToSettingsApp"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectProgramsAndFeaturesToSettingsApp"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectDateAndTimeToSettingsApp"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"DoNotRedirectNotificationIconsToSettingsApp"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdatePolicy"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdatePreferStaging"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdateAllowDowngrades"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdateURL"=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"UpdateURLStaging"=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"AllocConsole"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"Memcheck"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"TaskbarAutohideOnDoubleClick"=dword:00000000
[HKEY_CURRENT_USER\Control Panel\Desktop]
"PaintDesktopVersion"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ClassicThemeMitigations"=dword:00000000
[-HKEY_CURRENT_USER\Software\Classes\CLSID\{1eeb5b5a-06fb-4732-96b3-975c0194eb39}\InprocServer32]
@=""
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"NoPropertiesInContextMenu"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"EnableSymbolDownload"=dword:00000001
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"PinnedItemsActAsQuickLaunch"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"RemoveExtraGapAroundPinnedItems"=dword:00000000
[HKEY_CURRENT_USER\Software\ExplorerPatcher]
"ExplorerReadyDelay"=dword:00000000`

this problem occurs with default settings also

I use the old windows 98 Quick Launch and a single click will open the file explorer on my E: drive with a shortcut `%SystemRoot%\explorer.exe /e,E:`  this allows me to open multiple instances without changing the path of the first open window

**Reply by pyrates999** [reply]
Thank you.

**Comment by Jamesstevens-3rd** [comment]
looks like its fixed, Thank you for addressing this problem

/james
